README.md

```diff
@@ -186,6 +214,164 @@ grants-tagger train bertmesh \
+
+```
+==================================================
+The SD BIOLINE HIV/Syphilis Duo assay is the first World Health Organization prequalified dual rapid diagnostic test for
+ simultaneous detection of HIV and Treponema pallidum antibodies in human blood. Prior to introducing the test into 
+ antenatal clinics across South Sudan, a field evaluation of its clinical performance in diagnosing both HIV and 
+ syphilis in pregnant women was conducted. SD Bioline test performance on venous blood samples was compared with (i) 
+ Vironostika HIV1/2 Uniform II Ag/Ab reference standard and Alere Determine HIV 1/2 non-reference standard for HIV 
+ diagnosis, and (ii) Treponema pallidum hemagglutination reference standard and Rapid plasma reagin non-reference 
+ standard for syphilis. Sensitivity, specificity, positive predictive value (PPN), negative predictive value (NPV) 
+ and kappa (ê) value were calculated for each component against the reference standards within 95% confidence 
+ intervals (CIs); agreements between Determine HIV 1/2 and SD Bioline HIV tests were also calculated. Of 442 pregnant 
+ women recruited, eight (1.8%) were HIV positive, 22 (5.0%) had evidence of syphilis exposure; 14 (3.2%) had active 
+ infection. For HIV diagnosis, the sensitivity, specificity, PPV and NPV were 100% (95% CI: 63.1-100), 100% 
+ (95% CI: 99.2-100), 100% (95% CI: 63.1-100) and 100% (95% CI: 99.2-100) respectively with ê value of 1 
+ (95% CI: 0.992-1.000). Overall agreement of the Duo HIV component and Determine test was 99.1% (95% CI: 0.977-0.998) 
+ with 66.7% (95% CI: 34.9-90.1) positive and 100% (95% CI: 0.992-1.000) negative percent agreements. For syphilis, 
+ the Duo assay sensitivity was 86.4% (95% CI: 65.1-97.1) and specificity 100% (95% CI: 99.1-100) with PPV 100% 
+ (95% CI: 82.4-100), NPV 99.2% (95% CI: 97.9-99.9) and ê value 0.92 (95% CI: 0.980-0.999). Our findings suggest the SD Bioline HIV/Syphilis Duo Assay could be suitable for HIV and syphilis testing in women attending antenatal services across South Sudan. Women with positive syphilis results should receive treatment immediately, whereas HIV positive women should undergo confirmatory testing following national HIV testing guidelines.
+==================================================
+[2/100]> Is this  a `HIV` text? [a to accept]:
+
+```
+
+If not set, the model will randomly get *train-examples* and train the classifier without your supervision, which will reduce the performance of the classifiers.
+
+### Artifacts created
+As a result of the proces, you will find a folder at *save_to_path*. Inside,  you will find:
+- One folder per tag, including:
+  - `clf` (a classifier),
+  - `curation` (a dataset of positive and negative examples for the tag)
+  - `labelbinarizer` (a label binarizer to encode the labels)  
+- a `corrections` file, the new allMeSH_2021.jsonl with your tags corrected.
```

Comment:
all these not tracked in dvc i assume :( is it also 24GB each time?

Reply:
I think we only need to track the `corrections` file, which is the retagged mesh file. After correction, we don't really need the classifiers anymore.

Reply:
I have added the curation files for each of the 5 tags too just in case. BTW it's not 24GB, that's all mesh. We are using only years 2016-2021, which is 2M rows, about 5GB.